artists.css

```diff
@@ -0,0 +1,163 @@
+@import url("https://fonts.googleapis.com/css2?family=Explora&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Times New Roman", Times, serif;
+}
+body {
+  background-image: linear-gradient(to right, #000000, #130f40, #000000);
+  min-height: 100vh;
+}
+
+.nav-bar {
+  position: fixed;
+  width: 100%;
+  background-color: black;
+  display: flex;
+  justify-content: space-around;
+}
+.nav-bar img {
+  max-width: 18rem;
+  max-height: 5.4rem;
+  padding-inline-start: 0.3rem;
+}
+.nav-bar nav {
+  display: flex;
+  flex-direction: column;
+}
+.menu-items .menu-item {
+  text-decoration: none;
+  color: white;
+  padding-block: 0.1rem;
+  font-weight: 800;
+}
+.menu-item:hover {
+  transform: scale(110%);
+}
+.wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, auto);
+  width: 100%;
+  padding-block-start: 7rem;
+}
+.container {
+  grid-column: 1 / 2;
+  width: 100%;
+  height: 40%;
```

Comment:
Elementele din containerul tău par să fie mereu mai mari decât containerul. Ai putea scoate înălțimea de aici. Cred că și înălțimea.

Reply:
Am scris greșit. Voiam să scriu că se poate scoate și lățimea (width)